sphinx/source/docs/dev_guide/setup.rst

```diff
@@ -249,12 +249,32 @@ which include:
 * pytest-selenium >= 1.0
 * mock
 * websocket-client
+* flake8
+* boto
 
 Both the build and test dependencies can potentially change between releases
 and be out of sync with the hosted Bokeh site documentation, so the best way
 to view the current required packages is the review the meta.yaml_ file included
 in the Github repository.
 
+In addition to the build and test dependencies, you must also have the base
+dependencies for Bokeh installed. A simple way to install these dependencies
+is to install Bokeh via ``conda install`` or ``pip install`` before running
+``setup.py``.  Alternatively, you can download them indivually. The
+dependencies include:
+
+* backports_abc
+* jinja2
+* markupsafe
+* mkl
```

Comment:
like the other PR, `mkl` is absolutely 100% not a required dependency. Conda will install it by default for any numpy install, but that has nothing to do with Bokeh.